packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -62,6 +63,42 @@ export class OpenPrimitiveElementOpcode extends Opcode {
   }
 }
 
+export class PushRemoteElementOpcode extends Opcode {
+  public type = "push-remote-element";
+
+  evaluate(vm: VM) {
+    // This is asserting that the operand is an Element.
+    // But how do you know?
+
+    // {{#-with-element "zomg"}} <-- you'll get a string back here, now what?
+    let reference = vm.frame.getOperand<Simple.Element>();
```

Comment:
For future readers: this is unsafe but we decided to punt on error-ing since this is still experimental